universal-application-tool-0.0.1/app/views/admin/questions/QuestionEditView.java

```diff
@@ -38,31 +40,47 @@
 public final class QuestionEditView extends BaseHtmlView {
   private final AdminLayout layout;
 
+  private static final String NO_REPEATER_DISPLAY_STRING = "applicant";
```

Comment:
Suggestion (blocking): 
```suggestion
  private static final String NO_REPEATER_DISPLAY_STRING = "does not repeat";
```